<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scoreboard</class>
 <widget class="QWidget" name="Scoreboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="ControlWidget" name="widgetControlLeft" native="true">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TimerWidget" name="widgetTimer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ControlWidget" name="widgetControlRight" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlWidget</class>
   <extends>QWidget</extends>
   <header>controlwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimerWidget</class>
   <extends>QWidget</extends>
   <header>timerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
